_guide/_pages/interviews/core-values.md

```diff
@@ -54,108 +54,61 @@ Begin by introducing yourself by saying this or something similar to it:
-
-### Technology and open source
-
-Regardless of whether or not the candidate is applying for a software development position, 18F relies heavily on technology and open source to execute its mission. This section focuses on those skills and experience.
-
-- [Have you ever worked with open source before?](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.3qcfg9pfhobk)
-    - If applicable: What were you involved with, and what was your role?
-    - If applicable: How did you contribute?
-
-- [Are you comfortable with sharing your work in a public fashion?](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.3qcfg9pfhobk)
-
-- [Have you ever advocated for open source in a previous job?](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.3qcfg9pfhobk)
-    - If applicable: Were you successful in doing so, and why or why not?
-
-- [What is your comfort level with peer-reviewed work?](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.3qcfg9pfhobk)
-    - If applicable: Have you proactively sought additional training and/or peer review?
-
-- [How would you deal with a bug in an open source project?](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.3qcfg9pfhobk)
 
 ### Diversity, equity, and inclusion
 
 Diversity, equity, and inclusion are core values for 18F and the engineering chapter. These questions are meant to shed light on the candidate's experience and thoughts.
 
-- "At 18F we strive to be as representative of the populace as possible and work hard to ensure that everyone understands this notion and is treated fairly and respectfully. [What do diversity, equity, and inclusion mean to you?](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.5kwvj8b9hvhd)"
+- "[Tell me about a time you’ve worked with or for communities different than your own. What did you learn?](https://docs.google.com/document/d/1Z9A0WJFJj_Vo975RkuzQQazEOSJTPixiiahrtlNOBiI/edit#heading=h.u30z4bws4grp)"
+    - What do diversity and inclusion mean to you?
+    - How do you take equity into consideration in your software practice?
 
 ### Overall experience and outlook
 
 Begin closing the interview by asking general questions to ensure we develop a more well-rounded picture of the candidate. These questions aren’t geared toward a specific job or work methodology. Instead, they’re meant to shed light into the candidate’s overall experience and outlook.
 
 Ask one or more of the following:
 
-- [Tell me about a time that something at work didn’t go well.](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.6e8l8bhmhbi3)
```

Comment:
This is a question I find _very_ valuable when interviewing candidates (for any role) - being able to acknowledge your past mistakes and describe what you learned from them / how you avoided similar issues in the future is a pretty important trait, IMO. I'd go so far as to say that if a candidate can't describe a single situation where something went wrong at work that they were able to learn from (i.e. everything's always been perfect) then that's a red flag for me.

I'd normally pair it with the `what you've done that you're most proud of` question, but if I had to pick one I'd pick this one since it directly speaks to humility and integrity.